test/fn/watch-project.spec.js

```diff
@@ -141,6 +141,13 @@ describe('watch-project', function () {
   });
 
   it('watch-project: upload custom translations', () => {
+    api.giveResponses(
+      {
+        status: 200,
+        body: { version: '3.5.0' },
+      },
+    );
```

Comment:
It times out locally without this mocked response, my guess is it's because it calls `const version = await getValidApiVersion();` on line 171 https://github.com/medic/cht-conf/blob/575-retry-on-failure/src/fn/upload-custom-translations.js#L171